docs/getting-started/aspnet5/sample/src/EFGetStarted.AspNet5/Views/Blogs/Index.cshtml

```diff
@@ -0,0 +1,30 @@
+﻿@model IEnumerable<EFGetStarted.AspNet5.Models.Blog>
+
+@{
+    ViewBag.Title = "Blogs";
+}
+
+<h2>Blogs</h2>
+
+<p>
+    @Html.ActionLink("Create New", "Create")
```

Comment:
Ultimately this will be replaced by scaffolding... so I think I'll just leave it as-is for the moment and before RTM the code will swap to whatever gets scaffolded by MVC.